idl/storage_service.idl.hh

```diff
@@ -76,5 +76,5 @@ verb [[cancellable]] tablet_stream_data (raft::server_id dst_id, locator::global
 verb [[cancellable]] tablet_cleanup (raft::server_id dst_id, locator::global_tablet_id);
 verb [[cancellable]] table_load_stats_v1 (raft::server_id dst_id) -> locator::load_stats_v1;
 verb [[cancellable]] table_load_stats (raft::server_id dst_id) -> locator::load_stats;
-verb [[cancellable]] tablet_repair(raft::server_id dst_id, locator::global_tablet_id) -> service::tablet_operation_repair_result;
+verb [[cancellable]] tablet_repair(raft::server_id dst_id, locator::global_tablet_id, service::frozen_topology_guard topo_guard [[version 2025.1]]) -> service::tablet_operation_repair_result;
```

Comment:
No need to send the topo guard here. The receiver side can get the session id with trinfo->session_id.